### PR TITLE
added list of community members (Community member list #31)

### PR DESCRIPTION
**Pull request by naoe-tatara**
Relevant issue: Community member list #31

### CODE REVIEW

**Comment by naoe-tatara** [comment]
Also List of past workshops #32

**Comment by lexnederbragt** [comment]
Looks good to me!

File: content/about/members.md [hunk 1]
```diff
@@ -25,6 +25,24 @@ Naoe Tatara
...
   _appointed by University of Oslo library_
 
-## Instructors
-
-We have many active instructors. Would you like to become an instructor, too? Join our mailing list or meet us at a study group.
\ No newline at end of file
+## Community members
+
+We have many Carpentries certified instructors, too. 
+Would you like to become an instructor, too? Join our mailing list or meet us at a study group.
```
Comment (Teebusch) [inline content/about/members.md]: maybe add the mailing list address or a link where to find it?

Comment (Teebusch) [reply content/about/members.md]: ... perhaps also a link to the study group page of the site ` [study group]({{< relref "/study_group" >}})`

Comment (naoe-tatara) [reply content/about/members.md]: That's true (I thought about that on Saturday and forgot today). I will fix that part and commit soon.

**Comment by naoe-tatara** [comment]
@Teebusch , I committed the change :)

**Review by Teebusch — APPROVED** [review]
looks good. can be merged, I think

**Pull request merged**